core-server/src/main/java/org/glassfish/jersey/server/internal/scanning/PackageNamesScanner.java

```diff
@@ -316,4 +323,13 @@ private String toExternalForm(final URL u) {
         }
         return result.toString();
     }
+
+    private static final Collection<UriSchemeResourceFinderFactory> defaultFinderFactories() {
```

Comment:
I suppose here you were going to declare a constant but ended-up with private static final method instead. Please decide whether that shall be a constant (private static final) or method (private).

Reply:
I've declared this into a method because the Data Structure might change. Personally i don't like having for example `new LinkedList()` in the constant field.